app/controllers/grant_submissions/submissions/unsubmit_controller.rb

```diff
@@ -8,7 +8,7 @@ def update
         if @submission.draft?
           flash[:warning] = 'This submission is already editable.'
         elsif @submission.update(state: GrantSubmission::Submission::SUBMISSION_STATES[:draft])
-          flash[:success] = 'You have changed the status of this submission to Draft. It may now be edited by the applicant.'
+          flash[:success] = 'You have changed the status of this submission to Draft. It may now be edited by the submitter.'
```

Comment:
This seems like a case where the code's language is seeping into the UI. From the grant admin perspective, it's editable by the applicant, no? Maybe just change to "It may now be edited."